replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/ReplicateRecords.scala

```diff
@@ -72,11 +72,13 @@ private[journal] object ReplicateRecords {
           def measure(events: Nel[EventRecord[EventualPayloadAndType]], expireAfter: Option[ExpireAfter]) = {
             for {
               measurements <- measurements(records.size)
-              version       = events.last.version.map(_.value).getOrElse("none")
+              version       = events.head.version.map(_.value).getOrElse("none")
+              expiration    = events.head.metadata.payload.expireAfter.map(_.value.toString).getOrElse("none")
```

Comment:
I think using _first_ event is not accurate because KJ will take expiration for _last_ event as final.

Reply:
yeah, and cutting of polls can happen at any place - so eventually we will cover all values also with `last`, but dominating values will come first - I will change